sql/field.cc

```diff
@@ -2240,7 +2240,6 @@ Field *Field::make_new_field(MEM_ROOT *root, TABLE *new_table,
                 VERS_SYS_START_FLAG | VERS_SYS_END_FLAG |
                 VERS_UPDATE_UNVERSIONED_FLAG);
   tmp->reset_fields();
-  tmp->invisible= VISIBLE;
```

Comment:
Did you test with `main.invisible_binlog main.invisible_field_debug main.invisible_field`?

Reply:
That was incorrect fix. I've read `git blame` incorrectly. Thanks for noticing.